service/src/main/java/greencity/service/ubs/ViolationServiceImpl.java

```diff
@@ -180,13 +182,13 @@ public void deleteViolation(Long id, String uuid) {
 
     @Override
     public void updateUserViolation(UpdateViolationToUserDto add, MultipartFile[] multipartFiles, String uuid) {
-        User currentUser = userRepository.findUserByUuid(uuid)
+        Employee currentUser = employeeRepository.findByUuid(uuid)
             .orElseThrow(() -> new UserNotFoundException(USER_WITH_CURRENT_ID_DOES_NOT_EXIST));
```

Comment:
please add test for case when  `.orElseThrow(() -> new UserNotFoundException(USER_WITH_CURRENT_ID_DOES_NOT_EXIST));`

Reply:
it was already added